### PR TITLE
Minor update

### PR DESCRIPTION
**Pull request by 5hu5ky**

### CODE REVIEW

File: Hack-A-Sat 2021/Rapid Unplanned Disassembly/Grade F Prime Beef/grade_f_write_up.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ PWD=/home/space/fprimeLOGNAME=spaceHOME=/home/spaceFLAG=flag{sierra457871juliet2
 _XXXXXXXXXXXXXXXX_XXXXXXXXXXXXXXXXXXXXXXXX_XXXXXXXXXXXXXXXX}
 USER=spaceSHLVL=0PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/gamesMAIL=/var/mail/spaceOLDPWD=/home/space_=./sa
 tellite.exe
-```
+```
```
Comment (mossmann) [inline Hack-A-Sat 2021/Rapid Unplanned Disassembly/Grade F Prime Beef/grade_f_write_up.md]: Not a big deal, but this one character change doesn't do anything.

Comment (mossmann) [inline Hack-A-Sat 2021/Rapid Unplanned Disassembly/Grade F Prime Beef/grade_f_write_up.md]: Did you mean to put the following couple commands (lines 157-158) in a code block?

**Review by mossmann — CHANGES_REQUESTED** [review]
Could you also please rename this file to README.md so that it is easier to navigate to on GitHub?

File: Hack-A-Sat 2021/Rapid Unplanned Disassembly/Grade F Prime Beef/grade_f_write_up.md (hunk 1 above)
Comment (5hu5ky) [reply Hack-A-Sat 2021/Rapid Unplanned Disassembly/Grade F Prime Beef/grade_f_write_up.md]: Markdown on my IDE acts funny - committed the changes.

**Review by mossmann — APPROVED** [review]
Excellent. Thanks!

**Pull request merged**